src/metatensor/models/experimental/alchemical_model/model.py

```diff
@@ -70,31 +70,31 @@ def forward(
         )
 
         total_energies: Dict[str, TensorMap] = {}
-        for output_name in outputs:
-            keys = Labels(
-                "_", torch.zeros((1, 1), dtype=torch.int32, device=predictions.device)
-            )
-            properties = Labels(
-                "energy",
-                torch.zeros((1, 1), dtype=torch.int32, device=predictions.device),
-            )
-            samples = Labels(
-                names=["system"],
-                values=torch.arange(
-                    len(predictions),
-                    device=predictions.device,
-                ).view(-1, 1),
-            )
-            block = TensorBlock(
-                samples=samples,
-                components=[],
-                properties=properties,
-                values=predictions,
-            )
-            total_energies[output_name] = TensorMap(
-                keys=keys,
-                blocks=[block],
-            )
+        output_name = list(outputs.keys())[0]  # only one output
```

Comment:
If only the energy is supported here, this should be 

```suggestion
        assert list(outputs.keys()) == ["energy"]
        output_name = "energy"
```